packets.go

```diff
@@ -525,6 +532,13 @@ func (mc *mysqlConn) handleErrorPacket(data []byte) error {
 		pos = 9
 	}
 
+	//If error code is for Connection was killed, then return bad connection.
+	//https://mariadb.com/kb/en/mariadb/mariadb-error-codes/
+	if errno == 1927 {
+		errLog.Print("Error ", errno, ": ", string(data[pos:]))
+		return driver.ErrBadConn
```

Comment:
my question is assuming this came after a query is sent. is `ErrPktSync` a recoverable error, i.e. can i still use the connection after I get back that error from the driver ?